src/tools/quemap/main.c

```diff
@@ -19,10 +19,10 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
  */
 
-#include <SDL2/SDL.h>
-
 #include "quemap.h"
 
+#include <SDL2/SDL.h>
+
```

Comment:
This file was a special exception. I needed config.h to be included to be able to tell SDL to stop trying to get SDL_main working.. it's an unconditional include on Windows, even if you want to handle WinMain yourself (as I do in winmain.c)